core/variant/binder_common.h

```diff
@@ -78,7 +78,7 @@ struct VariantCaster<const T &> {
 		_FORCE_INLINE_ static m_enum convert(const void *p_ptr) {            \
 			return m_enum(*reinterpret_cast<const int64_t *>(p_ptr));        \
 		}                                                                    \
-		typedef int64_t EncodeT;                                             \
+		using EncodeT = int64_t;                                             \
 		_FORCE_INLINE_ static void encode(m_enum p_val, const void *p_ptr) { \
 			*(int64_t *)p_ptr = p_val;                                       \
 		}                                                                    \
```

Comment:
Is `EncodeT` dead code? It's not used before it goes out of scope.

Reply:
No, it's used in `gdvirtual.gen.inc` generated by `core/object/make_virtuals.py`.

Reply:
I added a comment above the `#define` to clarify that EncodeT is being used in other files.